sqlx_todo/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "sqlx_todo"
+version = "0.1.0"
+authors = ["Milan Zivkovic <zivkovic.milan@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+listenfd = "0.3.3"
+actix-web = "3.0.0-alpha.1"
```

Comment:
Hm, I'd like to stay v2 for a while since v3-alpha is unstable. Does this example require v3?

Reply:
@JohnTitor I will check it now. I have used ctrl+space to fill it automatically in CLion and didn't give it too much attention... I will check it and push update... thanks

Reply:
Done